apps/expo-go/ios/Podfile.lock

```diff
@@ -3472,7 +3472,7 @@ SPEC CHECKSUMS:
   GoogleDataTransport: aae35b7ea0c09004c3797d53c8c41f66f219d6a7
   GoogleMaps: 80ea184ed6bf44139f383a8b0e248ba3ec1cc8c9
   GoogleUtilities: 26a3abef001b6533cf678d3eb38fd3f614b7872d
-  hermes-engine: 197827f1ec648b53dcdc503d7a1b3a7644f9b4cb
+  hermes-engine: b2efca291af9785edd0128611dbc1b1fb4020a6a
```

Comment:
Are these changes necessary? Can you delete the pods and install again?

Reply:
@alanjhughes it does come creeping back

Reply:
The hermes change seems correct, but not sure why glog and folly are changing @vonovak 🤔

Reply:
not sure what it was but now it's the hash is as before